tests/ui/conftest.py

```diff
@@ -1,37 +1,85 @@
 import datetime
 import json
 import os
+import random
 import urlparse
 
 import jwt
 import pytest
 import requests
 from django.core.management import call_command
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
-from olympia.amo.tests import create_switch
+from olympia import amo
```

Comment:
All of the `from olympia` imports will fail without the dev requirements installed, and if running against a deployed instance such as https://addons.allizom.org. Let's take care of this in a separate pull, but please raise an issue and add a TODO so we don't drop this.